Automattic/wp-calypso
# Pricing Grid: Add popular badge

#### Proposed Changes

* This PR is one among several stacked PRs that implements the new pricing grid design. In https://github.com/Automattic/wp-calypso/pull/71372, we introduced a new signup flow /start/onboarding-2023-pricing-grid that is only available in dev and wpcalypso environments.
* The design for the pricing grid can be found in 2e991-pb. Please keep the Figma design open in a separate tab.
* The parent PR is https://github.com/Automattic/wp-calypso/pull/71480.
* This PR adds the styling changes for highlighting the Premium plan as popular.

<img width="1548" alt="Screenshot 2022-12-26 at 12 50 51 PM" src="https://user-images.githubusercontent.com/1269602/209517403-f047e82f-e56c-428f-b9ef-aec9642529e2.png">

#### What Not To Test

* Responsive screens. It will be implemented in a separate PR. Right now, test in a full width desktop browser viewport.
* Non-en. The first launch will be `en` only.
* Verify UI changes only in the plan header section (everything up to the CTA buttons). The plan features will be styled in subsequent PRs.

#### Testing Instructions



* Navigate to `/start/onboarding-2023-pricing-grid`.
* Verify that the Premium plan is highlighted as popular, and that the styling matches the design in Figma.
* Verify that the production `onboarding` flow at `/start` is unaffected by UI changes.
